Modules/_bisectmodule.c

```diff
@@ -44,56 +77,79 @@ internal_bisect_right(PyObject *list, PyObject *item, Py_ssize_t lo, Py_ssize_t
     return lo;
 }
 
+/*[python input]
+class hi_parameter_converter(CConverter):
+    type = 'Py_ssize_t'
+    converter = 'optional_ssize_t_converter'
+
+[python start generated code]*/
+/*[python end generated code: output=da39a3ee5e6b4b0d input=53c711121eb30d3f]*/
+
+
+/*[clinic input]
+bisect.bisect_right
```

Comment:
You can use the return converter for simplifying the code.
```
bisect.bisect_right -> Py_ssize_t
```

The body of `bisect_bisect_right_impl()` would be just:
```
return internal_bisect_right(a, x, lo, hi);
```